### ISSUE TITLE
No error reported

### ISSUE THREAD

**Issue by smutel** [reviewdog/action-golangci-lint#803]
Hello,

**Here is my github action configuration:**

```yaml
- name: Lint                                                              
  uses: reviewdog/action-golangci-lint@f9bba13753278f6a73b27a56a3ffb1bfda90ed71  # v2.8.0
  with:                                                                   
    fail_level: any                                                       
    github_token: ${{ secrets.github_token }}                             
    golangci_lint_flags: "--config=.golangci.yml"
```

**Here is the result in the CI:**

*golangci-lint:*

```
Running golangci-lint ...
  /home/runner/work/_temp/reviewdog-NhurSq/golangci-lint-2.1.6-linux-amd64/golangci-lint run --output.text.path stdout --config=.golangci.yml
  netbox/internal/customfield/customfield.go:1:1: package-comments: should have a package comment (revive)
  package customfield
  ^
  netbox/internal/customfield/customfield.go:22:5: exported: exported var CustomFieldSchema should have comment or be unexported (revive)
  var CustomFieldSchema = schema.Schema{
      ^
...
  50 issues:
  * revive: 50
```

*reviewdog:*

```
Running reviewdog 🐶 ...
  /home/runner/work/_temp/reviewdog-ZX2lgr/reviewdog -f=golangci-lint -name=golangci -reporter=github-pr-check -filter-mode=added -fail-level=any -fail-on-error=false -level=error
  time=2025-05-08T15:57:26.323Z level=INFO msg="reviewdog: fallback to use git command"
```

But not error reported in the CI. What I miss ?
